src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/facebook/FacebookSettingsPageObject.java

```diff
@@ -32,7 +32,7 @@
   private WebElement settingsContent;
   @FindBy(css = "#userNavigationLabel")
   private WebElement fbDropDown;
-  @FindBy(css = ".uiLinkButtonInput")
+  @FindBy(xpath = "//*[contains(@class, 'uiContextualLayer')]//span[text()='Log out']")
```

Comment:
```xpath = "//*[contains(@class, 'uiContextualLayerBelowRight')]//span[text()='Log out']"``` would be even better selector in this case, because there is only one element with 'uiContextualLayerBelowRight' class and three elements with 'uiContextualLayer' class. So in case there will be other element with text "Log out", it will pass.